graphics/py-pydicom/Makefile

```diff
@@ -0,0 +1,27 @@
+# $NetBSD$
+
+DISTNAME=	pydicom-1.2.0
+PKGNAME=	${PYPKGPREFIX}-${DISTNAME}
+CATEGORIES=	graphics python
+MASTER_SITES=	${MASTER_SITE_GITHUB:=pydicom/}
+GITHUB_PROJECT=	pydicom
+GITHUB_TAG=	v${PKGVERSION_NOREV}
+
+MAINTAINER=	eborisch@gmail.com
+HOMEPAGE=	https://github.com/pydicom/pydicom/
+COMMENT=	Pure python package for working with DICOM files
+LICENSE=	modified-bsd AND mit
+
+WRKSRC=	${WRKDIR}/pydicom-1.2.0
```

Comment:
Does it still work if you drop this line?

Reply:
Yes. Done.